experimenter/experimenter/nimbus-ui/src/components/AppLayoutSidebarLaunched/index.test.tsx

```diff
@@ -105,7 +105,9 @@ describe("AppLayoutSidebarLaunched", () => {
         `${BASE_PATH}/my-special-slug`,
       );
 
-      screen.getByText("Experiment analysis not ready yet.", { exact: false });
+      screen.getByText("Analysis results are not available for rollouts.", {
+        exact: false,
+      });
```

Comment:
And then you'd want two versions of this test, one for the rollout case, one for the experiment case.